packages/ticketing/src/optional.d.ts

```diff
@@ -0,0 +1,20 @@
+// src/types/optional.d.ts
+declare module '@react-pdf/renderer' {
```

Comment:
That file is not needed, put those dependencies into dev dependences of ticketing and optional dependencies, that way you can access the types safely.